data/strings/sound.txt

```diff
@@ -426,9 +499,135 @@
     tr = çıkın.
     uk = З'їзд.
     vi = Lối thoát.
-    zh-Hans = 出口.
+    zh-Hans = 驶出
     zh-Hant = 出口.
 
+  [onto]
+    comment = Used in dist_direction_onto_street in position 3 unless take_exit_number or onto_exit_number are specified.
+    en = onto
+    ar = إلى
+    be = на
+    cs = na
+    da = ad
+    de = auf
+    el = στην
+    es = hacia
+    es-MX = hacia
+    eu =
```

Comment:
Are empty strings expected? If yes, then it's better to mention in a comment.

Reply:
Some languages don't appear to have an "onto" word so yes sometimes null is possible. I will update the comment.

Reply:
Tested and commented